hibernate-types-43/src/main/java/com/vladmihalcea/hibernate/type/json/internal/JsonNodeTypeDescriptor.java

```diff
@@ -15,6 +18,17 @@
 
     public JsonNodeTypeDescriptor() {
         super(JsonNode.class, new MutableMutabilityPlan<JsonNode>() {
+
+            @Override
+            public Serializable disassemble(JsonNode value) {
```

Comment:
Looks good. The `MutabilityPlan` is the one who takes care of the marshalling/unmarshalling. That's why these two methods are final in the Hibernate Type.